src/legacy/core_plugins/kibana/public/discover/index.ts

```diff
@@ -16,24 +16,5 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import chrome from 'ui/chrome';
-import { PluginInitializerContext } from 'kibana/public';
-import { npSetup, npStart } from 'ui/new_platform';
-import { DiscoverPlugin } from './plugin';
-
-// Core will be looking for this when loading our plugin in the new platform
-export const plugin = (context: PluginInitializerContext) => {
```

Comment:
Just a quick question: shouldn't `plugin` stay in `index.ts` be imported in legacy.ts? since "core will be looking for this when loading our plugin in the new platform". it that's the case index.ts would already be ready for NP, wouldn't it be?
however it's not a big change to be done when the cutover is done

Reply:
You are completely right, changed that. A small thing, but will make final migration easier.